tablet-udev-gen-rules.sh

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/env bash
+# OpenTabletDriver udev rules generator, written in bash
```

Comment:
Where was this file taken from? Is there a link or attribution required?

Reply:
Let me explain, in Linux for a long time, distributions have stored binaries in different directories (/bin, /usr/bin, /sbin, /xbin, /usr/sbin,....) and it so happened that on some distributions the `bash` file  is not in /bin/bash, but elsewhere.  However, the `env` command, which runs another command, passing environment variables to it, most often lies in /usr/bin

Reply:
https://en.m.wikipedia.org/wiki/Env#Examples

Reply:
I'm referring to the whole file, not the specific line. Is it something you made?

Reply:
Yes. It's udev rule generator for graphics tablets. OTD uses evdev(userspace) to create devices in /dev/input/input*, and for that, `udev` rules requires.